src/browser/browser.js

```diff
@@ -36,6 +36,11 @@ define((require, exports, module) => {
     scrollGrab: VirtualAttribute((node, current, past) => {
       node.scrollgrab = current;
     }),
+    mouseInHotzone: VirtualAttribute((node, current, past) => {
+      if (current && !past) {
+        showNavbar();
```

Comment:
BTW `node` here is the `main` so you could pass it into `showNavBar` function to avoid using `querySelector`.